src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -231,6 +232,104 @@ BOOST_AUTO_TEST_CASE(JacobianConstantLayerTest)
 }
 
 /**
+ * Tests the BatchNorm Layer, compares the layers parameters with
+ * the values from another implementation.
+ */
+BOOST_AUTO_TEST_CASE(BatchNormTest)
+{
+  arma::mat dataset, output;
+  data::Load("iris.csv", dataset, true);
+
+  arma::mat input = dataset.submat(0, 0, dataset.n_rows-1, 2);
+
+  BatchNorm<> model(input.n_rows);
```

Comment:
It looks like this line causes the windows build error

```
3949: [00:07:04] C:\projects\mlpack\src\mlpack\tests\ann_layer_test.cpp(245): error C2664: 'mlpack::ann::BatchNorm<arma::mat,arma::mat>::BatchNorm(mlpack::ann::BatchNorm<arma::mat,arma::mat> &&)': cannot convert argument 1 from 'const arma::uword' to 'const mlpack::ann::BatchNorm<arma::mat,arma::mat> &' [C:\projects\mlpack\build\src\mlpack\tests\mlpack_test.vcxproj]
```

maybe if we cast ``input.n_rows`` to size_t it work's?